data/en/community.json

```diff
@@ -6,7 +6,7 @@
       "items": [
         {
           "title": "Ask a question",
-          "body": "You get in touch with the community using any of the following links:",
+          "body": " You get in touch with the community using any of the following channels:",
```

Comment:
remove unnecessary space in the beginning?

Reply:
done